tests/code/test_participants.py

```diff
@@ -9,10 +9,10 @@
 from app.models.program import Program
 from app.models.eventParticipant import EventParticipant
 from app.models.programEvent import ProgramEvent
-from app.logic.volunteers import addVolunteerToEventRsvp
 from app.logic.volunteers import getEventLengthInHours, updateEventParticipants
-from app.logic.participants import userRsvpForEvent, unattendedRequiredEvents, sendUserData, getEventParticipants, trainedParticipants, getUserParticipatedEvents
+from app.logic.participants import userRsvpForEvent, unattendedRequiredEvents, sendUserData, getEventParticipants, trainedParticipants, getUserParticipatedEvents, checkUserAddedToEvent
 from app.models.eventRsvp import EventRsvp
+from dateutil import parser
```

Comment:
Put the library imports together (as opposed to our app). An organizational thing